### ISSUE TITLE
Non-deterministic required version ordering

### ISSUE THREAD

**Issue by acdha** [segmentio/terraform-docs#234]
### Prerequisites

Put an `x` into the box that applies:

- [x] This issue describes a bug.
- [ ] This issue describes a feature request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

I ran into an unusual new behaviour recently with the latest build in homebrew:

```
terraform-docs version v0.0.0- v0.9.0 darwin/amd64 BuildDate: 2020-04-01T14:50:28+0100
```

#### Actual Behavior

Much of the time when I run `terraform-docs markdown .` it outputs this block:

```
## Requirements

| Name | Version |
|------|---------|
| terraform | ~> 0.12.24 |
| aws | ~> 2.55.0 |
| http | ~> 1.1.1 |
```

Some fraction of the time, however, it swaps the order:

```
## Requirements

| Name | Version |
|------|---------|
| terraform | ~> 0.12.24 |
| http | ~> 1.1.1 |
| aws | ~> 2.55.0 |
```

Since we have pre-commit as part of our CI process, this caused some confusing CI failures.

I'm assuming this is something like the versions being stored in a hashmap which does not guarantee ordering and the easiest answer would be to always sort it on output.

**Comment by khos2ow** [comment]
Thanks for reporting, this is duplicate of #231 and fixed in #233

**Comment by acdha** [comment]
Sorry about that - not sure why Github’s search failed to return that for “order” or “sort” but I’m glad it’s fixed. This is a really useful part of our workflow.
